components/patient-data/ui/src/main/resources/PhenoTips/PatientClass.xml

```diff
@@ -257,6 +257,19 @@
       <validationRegExp/>
       <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
     </date_of_death_entered>
+    <delivery_mode>
+      <customDisplay/>
+      <disabled>0</disabled>
+      <name>delivery_mode</name>
+      <number>47</number>
+      <picker>0</picker>
+      <prettyName>Delivery mode</prettyName>
+      <size>30</size>
+      <unmodifiable>0</unmodifiable>
+      <validationMessage/>
+      <validationRegExp/>
+      <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
```

Comment:
This should be a `StaticList`, with several options: NSVD, VAD, Forceps Delivery, Odón Device, Primary Cesarean Section, Secondary Cesarean Section, VBAC. @bgallinger is this correct?